### DISCUSSION TITLE
Cisco Inconsistent Model Names

### DISCUSSION THREAD

**Discussion by etherwrangler** [netbox-community/devicetype-library#1459] [Ideas]
I'm getting started on my first NetBox install and importing all of our Cisco switches and I've noticed that the Model field is very inconsistent across product lines.  Take for instance the C9200L, C2960X, and C3550.

C9200L-48P-4G
Catalyst 2960X-48FPD-L
WS-C3550-24-SMI

If there's not a reason for this I'd propose that every Model should be prefaced by the product line, e.g. Catalyst, Nexus, UCS, MDS, etc, just for consistency.

**Comment by danner26** [comment]
So the way the data is supposed to be modeled is based on the device's output. If the device reports itself as `WS-C3550-24-SMI`, then the submitter should make the model match that. If it reports something else, then it should match that. And finally if the device does not have an interface that reports it's model name then the submitter should make it match what the datasheet calls the device. 

With this all said, a lot of these files have been around since the early versions of NetBox before standardization was put in place. Also, there are cases where the maintainers are not able to validate data so the submitter is taken for their word. 

If you ever find an issue and have evidence regarding a correction, we highly appreciate all PRs to solve these issues! Please feel free to submit a PR and add the cli output/datasheet in the comments

Let me know if you have any other questions!

**Comment by etherwrangler** [comment]
So on any Meraki device we should be dropping the word "Meraki" from the model field since the Model in the Meraki Dashboard does not show it.

![image](https://github.com/netbox-community/devicetype-library/assets/7719442/ad05ccba-6bb9-4cb2-b94b-6bc992d6f71a)

Also on Cisco Nexus and Cisco Catalyst devices we'd drop Nexus and Catalyst from the model.  The `show inventory` off a Nexus 5548 UP doesn't show that as part of the PID and I can't find the similar on any of our switches from the Catalyst line (2960/3560/3650/3850/9200).
```
NAME: "Chassis", DESCR: "Nexus5548 Chassis"
PID: N5K-C5548UP       , VID: V01 , SN: 
```

Is that the intent?

**Comment by danner26** [comment]
This is where it gets really tricky. Over time this repo has evolved and it hasnt always had such strict guidelines. I think the best route forward now is to have the models match either A) the output from the CLI/Application interface about what it thinks its "model" is, or if that option is not available then B) the manufacturer's datasheet model. 

Where things get tricky is manufacturer product lines like the Merakis. The idea we have ran with so far is for manufacturer sub-prodct lines such as Meraki, PowerEdge, Catalyst, etc is to include the prodct line in the model name and then ensure that the `product_number` matches exactly (which is not set in stone, we can propose it to the community if there is a strong case). This is a tough situation and I am not sure if there is one solution that works best for all product lines.
